src/config/combo.json

```diff
@@ -588,85 +516,6 @@
 	  "jp": "入力フィールドが空の場合に表示される値を取得または設定します。$.ig.Combo.locale.placeHolder のオーバーライドです。"},
       "valueOptions": null
     },
-    "preventSubmitOnEnter": {
-      "type": "bool",
-      "defaultValue": true,
-      "description": {
-	  "en": "Gets sets ability to prevent form submitting on enter key press",
-	  "jp": "[Enter] キーを押すと、フォームの送信を回避する機能を取得または設定します。"},
-      "valueOptions": null
-    },
-    "requestType": {
-      "type": "string",
-      "defaultValue": "GET",
-      "description": {
-	  "en": "specifies the HTTP verb to be used to issue the request",
-	  "jp": "要求を送信する HTTP 動詞を指定します。"},
-      "valueOptions": null
-    },
-    "responseContentType": {
-      "type": "string",
-      "defaultValue": null,
-      "description": {
-	  "en": "content type of the response. See http://api.jquery.com/jQuery.ajax/ => contentType",
-	  "jp": "応答のコンテンツ タイプ。http://api.jquery.com/jQuery.ajax/ の contentType を参照してください。"},
-      "valueOptions": null
-    },
-    "responseDataKey": {
```

Comment:
@bazal4o I believe all these should be configured only on the data source level in the designer (the non-visual control configuration) and then the data source should be assigned to the control. This simplifies the usage, as currently the control doesn't guide you on how these should be configured.